variables.tf

```diff
@@ -10,6 +10,12 @@ variable "stage" {
   description = "Stage (e.g. `prod`, `dev`, `staging`)"
 }
 
+variable "use_fullname" {
+  type        = "string"
+  default     = "true"
+  description = "Set 'true' to use `cp-prod-bastion_image` for ecr repository name, else `bastion_image`"
```

Comment:
update to something like this:

Set to '"true"' to use the `namespace-stage-name` format for the ECR repository name, otherwise just the `name` will be used